libs/components/packages/package.json

```diff
@@ -95,6 +95,7 @@
     "@angular/core": "^17.1.1"
   },
   "dependencies": {
+    "cheerio": "0.22.0",
```

Comment:
This version introduces a package vulnerability, `nth-check`. Could we use the latest RC release instead?
https://github.com/cheeriojs/cheerio/releases/tag/v1.0.0-rc.12

Reply:
Updated.